src/urbit/app/chess.hoon

```diff
@@ -58,51 +70,73 @@
   |=  [=mark =vase]
   ^-  (quip card _this)
   ?+  mark  (on-poke:default mark vase)
+    ::
+    ::  pokes managing active game state and challenges
     %chess-action
+      ::  only allow chess actions from our ship or our moons
       ?>  (team:title our.bowl src.bowl)
       =/  action  !<(chess-action vase)
       ?-  -.action
+        ::  manage new outgoing challenges
         %challenge
+          ::  only allow one active challenge per ship
+          ::  TODO: change or display to frontend
           ?:  (~(has by challenges-sent) who.action)
             :_  this
             =/  err
               "already challenged {<who.action>}"
             :~  [%give %poke-ack `~[leaf+err]]
             ==
+          ::  send new challenge
           :-  :~  :*  %pass  /poke/challenge  %agent  [who.action %chess]
                       %poke  %chess-challenge  !>(challenge.action)
                   ==
               ==
+          ::  add to list of outgoing challenges
           %=  this
             challenges-sent  (~(put by challenges-sent) +.action)
           ==
         %accept-game
           =/  challenge  (~(get by challenges-received) who.action)
+          ::  if challenge is null...
           ?~  challenge
+          ::  TODO: this should display to frontend
             :_  this
             =/  err
               "no challenge to accept from {<who.action>}"
             :~  [%give %poke-ack `~[leaf+err]]
             ==
+          ::  if not, is our challenger
+          ::  happy to play as a random side?
```

Comment:
I can see this is the branch taken when our challenger is happy to play as random, and I see it's using entropy to generate hashes, but since I don't understand `%chess-rng` and `%commit` yet I'm going to just `XX:` this whole wutcol.